torchao/quantization/autoquant.py

```diff
@@ -232,14 +223,16 @@ def do_autoquant_bench(op, *args, **kwargs):
         graph = torch.cuda.CUDAGraph()
         with torch.cuda.graph(graph, stream=stream):
             op(*args, **kwargs)
-        if TORCH_VERSION_AFTER_2_3 and not TORCH_VERSION_AFTER_2_5:
-            from torch._inductor.runtime.runtime_utils import do_bench_gpu
-            res = do_bench_gpu(lambda: graph.replay(), warmup=warmup, rep=rep, return_mode="median")
-        elif TORCH_VERSION_AFTER_2_5 and torch.cuda.is_available():
+        if TORCH_VERSION_AFTER_2_4:
```

Comment:
if I understand this correctly, anything after 2.4 will have the BC changes (not 2.5)?